thesis.tex

```diff
@@ -4,18 +4,17 @@
 %
 % fullpageDraft      avoid the margins necessary for proper binding and
 %   just view or print a draft.
-% beforeDefense      makes the personal acknowledgements invisible;
+% beforeDefense      make the personal acknowledgements invisible;
 %   use this to print the copies you submit initially to the grad school
 %   for sending to the opponent panel, i.e. thesis readers (who shouldn't
 %   see those parts). For the final submission, after having successfully
 %   defended - drop this option.
-% noabbrevs          no notation & abbreviations list will be included
-%   in the thesis.
+% noabbrevs          avoid generation of a notation & abbreviations list
 %
 % Additionally, you must specify the degree for which you're writing
 % your thesis (MSc/PhD/MArch etc.)
 %
-\documentclass[PhD]{misc/iitthesis}
+\documentclass[PhD,fullpageDraft]{misc/iitthesis}
```

Comment:
Are you sure you wanted this change to be in the master branch? It doesn't seem related to glossary support.

Reply:
No, will revert this.